introspector/src/pc/mod.rs

```diff
@@ -249,6 +253,7 @@ impl ParachainCommander {
 								let to_tracker = trackers.entry(para_id).or_insert_with(|| {
 									let (tx, rx) = channel_with_capacities(collector::COLLECTOR_NORMAL_CHANNEL_CAPACITY, 1);
 									futures.push(ParachainCommander::watch_node_for_parachain(self.clone(), rx, para_id, api_service.clone()));
+									info!("Added tracker for parachain {}", para_id);
```

Comment:
We might probably add the similar log line if a parachain has been evicted (if it is not there yet of course).

Reply:
It already has a message on evicting.